README.md

```diff
@@ -69,17 +69,17 @@ nix-env -f /path/to/obelisk -iA command
 Note that `ob` will defer to the version found in your project's `.obelisk/impl` directory. To update that version specifically:
 
 ```shell
-ob thunk unpack ./obelisk/impl
-cd ./obelisk/impl
+ob thunk unpack ./.obelisk/impl
+cd ./.obelisk/impl
 # apply your changes
 ```
 
 If you want to commit your changes, first push them to your fork of obelisk and then
 
 ```shell
 cd /your/project/root
-ob thunk pack .obelisk/impl
-git add .obelisk/impl
+ob thunk pack ./.obelisk/impl
+git add ./.obelisk/impl
```

Comment:
Sure...but why... lol.

Reply:
`./` is implicit (except for calling executables), so these two lines were actually correct. I think the proper fix is making the above ones like these

Reply:
Done